package.json

```diff
@@ -57,6 +57,11 @@
     "rebuild": "npm run rebuild-changes",
     "build": "npm run build-changes",
     "build:arm": "npm run build-changes:arm",
+    "build:android": "npm ci --ignore-scripts; node scripts/build-android.js",
```

Comment:
I don't like having `npm ci` (nor `npm install`) in scripts, since this makes the scripts less re-usable. It basically just adds time spent redoing stuff (installing dependencies) if these scripts are called in sequence.

I believe it's better to install dependencies once and explicitly in the workflow.